p4-16/spec/P4-16-spec.mdk

```diff
@@ -6507,6 +6575,21 @@ are treated as keywords only in specific contexts (e.g., the keyword `actions`).
 | `typedef` | `varbit`  | `verify` | `void`       |
 |------|------|------|------|
 
+# Appendix: P4 reserved annotations { #sec-p4-reserved-annotations }
+
+The following table shows all P4 reserved annotations.
+
+|---------------|----------------------------------------------------|--------------------------|
+| Annotation    | Purpose                                            | See Section              |
++---------------+:---------------------------------------------------+:-------------------------+
+| `atomic`      | specify atomic execution                           | [#sec-concurrency]       |
+| `defaultonly` | actions can only appear in the default action      | [#sec-table-action-anno] |
+| `hidden`      | hides a controllable entity from the control plane | [#sec-name-annotations]  |
+| `name`        | assign local control-plane name                    | [#sec-name-annotations]  |
+| `pkginfo`     | assign package-level metadata properties           | [#sec-package-anno]      |
+| `tableonly`   | actions can only appear within a table             | [#sec-table-action-anno] |
```

Comment:
This table is missing the `@optional` annotation.

Reply:
yes, because the annotation is not yet in the spec, it is still just in a PR.